CanalTP/Hermod
# Configuration of mailer moved to variable in env.

#### What I did

I moved configuration of mailer in env.

#### How I did it

I updated `docker/php/init_vars.sh` to add `MAILER_TRANSPORT` `MAILER_HOST` `MAILER_USER` `MAILER_PASSWORD`
I removed `MAILER_HOST` from `composer.json` file in `env-map` section

#### How to verify it

I you build and run you should have all variables (above) in env.

I you set all variables in `docker/php/init_vars.sh` and you run this command: 
```
./bin/console hermod:report:send "YOUR_EMAIL"
```
You should receive report by email
